Update Obsidian Sync and third-party services.md (#372)
obsidianmd/obsidian-docs: en/Obsidian Sync/Obsidian Sync and third-party services.md

[[Introduction to Obsidian Sync|Obsidian Sync]] is a service for synchronizing your notes across devices. While it offers helpful features like [[Version history|restoring notes]], it's not designed to be used to back up your vault.

Though we recommend that you regularly back up your vault, be aware that **using a third-party syncing service together with Obsidian Sync may result in data corruption or data loss**.

If you set up the same third-party service on multiple devices, such as your phone, you risk having the service competing with Obsidian Sync whenever you make changes to your vault. This can lead to conflicts, duplicated files, or corrupted files.

To back up your vault using a third-party service, configure the service so that backups are only made from your primary device. For example, if you do most of your writing on your laptop, that's where you should configure your backups. 

> We suggest using dedicated backup services that do not automatically save new data to your local device. 

Reviewer: ```suggestion
We recommend using a dedicated backup service that doesn't automatically update data on your local device from your backup.
```

I'm not sure I understood the original sentence, so I might've misunderstood this.

- "recommend" is a stronger verb here depending on whether this is a practice we actively promote, or we're just making them aware of the practice.
- Typically you'd only use one backup service, right?
- Change to use contractions.

Author: These changes are great, thanks!
